src/connection.rs

```diff
@@ -39,7 +39,10 @@ where
     async fn batch_execute_async(&self, query: &str) -> ConnectionResult<()> {
         let diesel_conn = Connection(self.0.clone());
         let query = query.to_string();
-        task::spawn_blocking(move || diesel_conn.inner().batch_execute(&query))
+        task::spawn_blocking(move || {
+                async_bb8_diesel_query!(|| query.as_str());
+                diesel_conn.inner().batch_execute(&query)
```

Comment:
I assume that, to implement @rmustacc 's suggestion, this could be something along the lines of:

```suggestion
                async_bb8_diesel_query_start!(|| query.as_str());
                let res = diesel_conn.inner().batch_execute(&query);
                async_bb8_diesel_query_end!(|| query.as_str());
                res
```

Reply:
Yep, that's exactly right. I'll add this in a follow-up commit, depending on how my question to Robert shakes out.